buildroot-external/patches/occu/0170-WebUI-RegexSearchInDeviceLists.patch

```diff
@@ -0,0 +1,86 @@
+--- occu/WebUI/www/webui/webui.js.orig
++++ occu/WebUI/www/webui/webui.js
+@@ -7882,11 +7882,12 @@
+     //var patternList = m_value.split("|");
+     text            = text.toLowerCase();
+     
+-    for (var i = 0, len = patternList.length; i < len; i++)
+-    {
+-      if (0 <= text.indexOf(patternList[i])) { return true; }
+-    }
+-    
++    //for (var i = 0, len = patternList.length; i < len; i++)
```

Comment:
Commenting things just out is not appropriate. Please delete the lines instead that you think are definitly not required.

Reply:
Not resolved, see Line #12

Reply:
Sorry, my fault :)

Now it should be resolved

Reply:
Now, the original //conInfo... commented lines are removed, too?

Sometimes I use them for debugging

Reply:
hmm.. I don't know that this comments were from the original occu 🙈 

I could re-add them if necessary...